CHANGELOG.md

```diff
@@ -49,6 +49,55 @@ aws lambda invoke --function-name $PREFIX-data-migration1 \
 - (Optional) Update the `task-config` for all workflows that use the `sync-granule` task to include `workflowStartTime` set to
 `{$.cumulus_meta.workflow_start_time}`. See [here](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/sync_granule_workflow.asl.json#L9) for an example.
 
+##### After the `cumulus` deployment
```

Comment:
Two things for discussion:

This was originally intended to be included in the 11.0.0 release notes and have the 11.0.1 CHANGELOG simply note that it was an amended update to 11.0.1 and that all release notes for 11.0.0 should be followed and users should skip 11.0.0. 

With the team decision to also include additional master updates this muddies the waters a bit.   We can chose to *only* release 11.1.0 with it's *additional* changes but note that users should skip 11.0.0 and go direct to 11.1.0, or include this as a separate migration for 11.1.0.